download.md

```diff
@@ -6,6 +6,14 @@ title: Download
 <div class="columns">
   <div class="column">
 
+## Android and OpenHarmony Instructions
+
+For Android and OpenHarmony, most devices need to be in Developer
+Mode in order to sideload packages manually. More information and links
```

Comment:
Is this accurate? I don't think my Android phone is in developer mode and I can install the `apk`. I guess the special thing is that you need to give permissions to the browser to install apps from unknown origins, but that's not developer mode.

Reply:
Ah, you're right. I was thinking of the case where during development we use `adb` to install the apk, but for downloads the user can directly install it. I'll check with @jschwe on the text for OpenHarmony and raise a new PR for that. I'll remove this section for now.